src/redux/categories/categories.js

```diff
@@ -1,17 +1,33 @@
-import CHECK_STATUS from './actionTypes';
+import { createSlice } from '@reduxjs/toolkit';
+// import CHECK_STATUS from './actionTypes';
 
-const categoriesReducer = (state = '', action) => {
-  switch (action.type) {
-    case CHECK_STATUS:
-      return action.payload;
-    default:
-      return state;
-  }
+// const categoriesReducer = (state = '', action) => {
+//   switch (action.type) {
+//     case CHECK_STATUS:
+//       return action.payload;
+//     default:
+//       return state;
+//   }
+// };
+
+// export const checkStatus = () => ({
+//   type: CHECK_STATUS,
+//   payload: 'Under Construction',
+// });
```

Comment:
- [ ] Please refrain adding commented out code in your workspace as it is frowned upon and reduces maintainability and manageability. Kindly remove this code here.